src/utils/localization-utils.tsx/vacation-request-type-utils.tsx

```diff
@@ -0,0 +1,29 @@
+import { VacationType } from "generated/client";
+import strings from "localization/strings";
+
+/**
```

Comment:
Is this a typo?

Reply:
Yes, is a typo, removed.
```suggestion
/**
```